packages/rum-core/src/boot/rumPublicApi.spec.ts

```diff
@@ -385,4 +389,35 @@ describe('rum entry', () => {
       expect(errorSpy).toHaveBeenCalledTimes(3)
     })
   })
+
+  describe('addTiming', () => {
+    let addTimingSpy: jasmine.Spy<ReturnType<StartRum>['addTiming']>
+    let errorSpy: jasmine.Spy<() => void>
+    let rumGlobal: RumPublicApi
+    let setupBuilder: TestSetupBuilder
+
+    beforeEach(() => {
+      addTimingSpy = jasmine.createSpy()
+      errorSpy = spyOn(console, 'error')
+      rumGlobal = makeRumPublicApi(() => ({
+        ...noopStartRum(),
+        addTiming: addTimingSpy,
+        configuration: ({
+          isEnabled: () => true,
+        } as any) as Configuration,
```

Comment:
```suggestion
      const configuration: Partial<Configuration> = {
        isEnabled: () => true,
      }
      rumGlobal = makeRumPublicApi(() => ({
        ...noopStartRum(),
        addTiming: addTimingSpy,
        configuration: configuration as Configuration,
```

To help catch potential type issue for fields use in the configuration stub.
Same strategy could be applied in similar cases.